hazelcast/src/main/java/com/hazelcast/internal/server/tcp/TcpServerConnection.java

```diff
@@ -242,7 +242,9 @@ public boolean write(OutboundFrame frame) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(acceptorSide, connectionId);
+        int result = (acceptorSide ? 1 : 0);
```

Comment:
```suggestion
        int prime = 31;
        int result = 1;
        result = prime * result + (acceptorSide ? 1231 : 1237);
```

Shouldn't this use the same value as Boolean.hashCode? Or even just calling that method?

Reply:
Although it is a matter of taste, I have updated the hashcode implementation to include your suggested change.